src/Uno.UI/UI/Xaml/Controls/SplitView/SplitView.mux.cs

```diff
@@ -0,0 +1,850 @@
+			this.OnPaneClosed();
+
+			// IFC_RETURN(DirectUI::ElementSoundPlayerService::RequestInteractionSoundForElementStatic(xaml::ElementSoundKind_Hide, this));
+		}
+	}
+
+	private Path CreatePolygonalPath(Grid hostElement, int numPoints)
+	{
+		Path path = new Path();
+
+		UIElementCollection children = hostElement.Children;
+		children.Add(path);
+
+		// Set a transparent brush to make sure it's hit-testable.
+		SolidColorBrush transparentBrush = new SolidColorBrush();
+		transparentBrush.Color = new Color(0, 0, 0, 0);
+		path.Fill = transparentBrush;
+
+		// Create the path geometry into which we'll add our rectangle figures.
+		PathGeometry pathGeometry = new PathGeometry();
+		path.Data = pathGeometry;
+
+		PathFigureCollection figures = pathGeometry.Figures;
+
+		PathFigure figure = new PathFigure();
+
+		figures.Add(figure);
+
+		figure.IsClosed = true;
+
+		PathSegmentCollection segments = figure.Segments;
+
+		// The number of segments we have is equal to the number of points we have minus 1
+		// because this is a closed figure.
+		for (int i = 0; i < (numPoints - 1); ++i)
+		{
+			LineSegment segment = new LineSegment();
+			segments.Add(segment);
+		}
+
+		// Uno TODO
+		// var uiElementCore = path;
+		// uiElementCore.SetAllowsDragAndDropPassThrough(true);
+
+		return path;
+	}
+
+	private void UpdatePolygonalPath(Path path, Point[] points)
+	{
+		Geometry geometry = path.Data;
+
+		PathGeometry pathGeometry = geometry as PathGeometry;
+
+		PathFigureCollection figures = pathGeometry.Figures;
+
+		PathFigure figure = figures[0];
+
+		figure.StartPoint = points[0];
+
+		PathSegmentCollection segments = figure.Segments;
+
+		for (int i = 1; i < points.Length; ++i)
+		{
+			// Subtract 1 because the first point in our array is set as the figure's
+			// start point.
+			PathSegment segment = segments[i - i];
+			global::System.Diagnostics.Debug.Assert(segment is LineSegment);
+
+			((LineSegment)segment).Point = points[i];
+		}
+	}
+
+	// Uno Specific to the end of this file
+	private void OnContentChanged(DependencyPropertyChangedEventArgs e)
+	{
+		SynchronizeContentTemplatedParent();
+	}
+
+	protected internal override void OnTemplatedParentChanged(DependencyPropertyChangedEventArgs e)
+	{
+		base.OnTemplatedParentChanged(e);
+
+		// This is required to ensure that FrameworkElement.FindName can dig through the tree after
+		// the control has been created.
+		SynchronizeContentTemplatedParent();
+	}
+
+	private void SynchronizeContentTemplatedParent()
+	{
+		// Manual propagation of the templated parent to the content property
+		// until we get the propagation running properly
+		if (Content is IFrameworkElement contentBinder)
+		{
+			contentBinder.TemplatedParent = this.TemplatedParent;
+		}
+		if (Pane is IFrameworkElement paneBinder)
+		{
+			paneBinder.TemplatedParent = this.TemplatedParent;
+		}
+	}
+
+	private bool IsGamepadNavigationDirection(VirtualKey key)
```

Comment:
These are duplicates from https://github.com/unoplatform/uno/blob/42f614e186e60ad4ebd6eb3934fe2ca74b1b70e1/src/Uno.UI/UI/Xaml/Controls/CommandBar/CommandBar.Partial.cs#L1367

Let's move this to a new static class `XboxUtility`